src/components/ref-example/ref-example.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types, jsx-a11y/no-static-element-interactions */
```

Comment:
Same as the "private" component comment, this is "private" in terms of the MDS site. It is only used as example interfaces. I figured we could relax the linters for these.

Reply:
Should we move it into the `__internal__` directory then?

Reply:
Nah, it's fine. React Styleguidist is configured to skip components without a `.md` file.